.github/workflows/Deploy.yaml

```diff
@@ -24,21 +30,28 @@ defaults:
 jobs:
   Deploy:
     runs-on: [ ubuntu-latest ]
+    environment: Production
     steps:
-      - uses: actions/checkout@v3
+      - name: Validate Deployment
+        if: github.repository_owner == 'microsoft'
+        env: 
```

Comment:
Do we need code for this? Can't this approval just be setup on the environment?

Reply:
We don't need it no :) And I'm open to removing it.

Without it, any of us would be able to Deploy, Approve and Merge changes to Al-Go preview / release branches on our own. 

GitHub does have an environment setting that lets you require approval from a list of approvers, but anyone in that list can also approve their own deployments. This might be okay though.. The "Validate Deployment" was just a way to force a "Sign-off" on the deployment from **someone else** before it deploys. 

Let me know what you think. I'm okay with either keeping it or removing it.

Reply:
Let's keep it in - but not require it when deploying to preview...